app/styles/app.scss

```diff
@@ -87,3 +87,22 @@ body {
 .btn btn-danger, btn-primary, a, hover {
   color: $white;
 }
+
+.item{
+    background: #333;    
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces